### PR TITLE
add russian locale

### PR DESCRIPTION
**Pull request by Awilum**
This code add the ru (russian) translation in the i18n support!

### CODE REVIEW

**Review by katherinepeterson — CHANGES_REQUESTED** [review]
Hey @Awilum, looks like when I click Russian on the dropdown, the webpage breaks.

Take a look at the deploy preview: https://readme-so-295jgbng1-katherinepeterson.vercel.app/

**Comment by Awilum** [comment]
> Hey @Awilum, looks like when I click Russian on the dropdown, the webpage breaks.
> 
> Take a look at the deploy preview: https://readme-so-295jgbng1-katherinepeterson.vercel.app/

seems it is my fault here: https://github.com/katherinepeterson/readme.so/pull/76/files#diff-00cf247b08806ad444456b54169f8eb5ca165eeb3f2c6bf7efbb027ef7026214R47
should be `/ru`

File: public/locales/ru/common.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "description": "Наш простой редактор позволяет быстро добавлять и настраивать все необходимые разделы для readme вашего проекта.",
+  "get-started": "Начать",
+  "100-free": "100% бесплатный",
+  "made-with-love": "Сделано с ",
+  "by": " от",
+  "language": "Язык",
+  "english": "Английский",
+  "french": "Французский",
+  "spanish": "Испанский",
+  "russian": "Русский",
```
Comment (rohitjmathew) [inline public/locales/ru/common.json:13]: @Awilum you seemed to have missed Italian over here. Could please add that

**Review by rohitjmathew — COMMENTED** [review]
The code looks good 💯 👍 🍻

Just notices one small miss. Added it below

File: public/locales/ru/common.json (hunk 1 above)
Comment (Awilum) [reply public/locales/ru/common.json:13]: but it is added here in the merge: https://github.com/katherinepeterson/readme.so/pull/76/commits/1a66d08102add9d2708d2088498c434343e49435

Comment (rohitjmathew) [reply public/locales/ru/common.json:13]: I'm sorry @Awilum, I don't think I communicated the issue correctly. I was just trying to point out at the Russian translation for `Italian` was missing in your `/ru/common.json` file. 

So if you check the preview and choose Russian, the language drop down show `Italian` in English and not in Russian. Ideally this should fix the issue.

```suggestion
  "italian": "Итальянский",
  "russian": "Русский",
```

Comment (Awilum) [reply public/locales/ru/common.json:13]: I see.
done https://github.com/katherinepeterson/readme.so/pull/76/commits/92fe93f2f354b40036159eb18497f806e2852f76

**Pull request closed without merging**